src/openms/source/KERNEL/Feature.cpp

```diff
@@ -67,7 +67,7 @@ namespace OpenMS
   Feature::Feature(Feature&& feature) noexcept :
     BaseFeature(std::move(feature)),
     convex_hulls_(std::move(feature.convex_hulls_)),
-    convex_hulls_modified_(std::move(feature.convex_hulls_modified_)),
+    convex_hulls_modified_(feature.convex_hulls_modified_),
```

Comment:
this could be expensive... check

Reply:
this is just a boolean, there is no need to use `std::move`. However if we remove it, the code looks a bit inconsistent

Reply:
agree - I would say we leave the std::move